src/Combobox/index.js

```diff
@@ -401,8 +400,8 @@ const Combobox = ({
     <>
       <div
         className={cc(["nds-combobox", { "nds-combobox--active": isOpen }])}
-        {...getComboboxProps(triggerProps)}
         data-testid={testId}
+        {...triggerProps}
```

Comment:
`triggerProps` comes from our modal positioning engine hook, `useLayer`. 

We still need these props spread onto the parent because `useLayer` needs it as a positioning reference.